prime-router/docs/design/proposals/mapping-inventory-test-plan.md

```diff
@@ -0,0 +1,38 @@
+# Mapping Inventory Test Plan
+
+## Problem Statement
+As a ReportStream engineer, I want tests that will test the following statement: "The Universal Pipeline (UP) shall be
+able to transform valid NIST 2.5.1 messages losslessly to FHIR using the official HL7v2 mapping inventory as a guide. 
+UP shall be able to transform FHIR ELR messages to valid NIST 2.5.1."
+
+Currently, we have integration tests that test an unknown combination of parts of sender transforms, receiver 
+transforms, and mapping inventory rules. We want a clear and definitive way of knowing what is already tested, and we 
+want the mapping inventory testing to be thorough. This means testing not only that each piece of the message ends up 
+in the Primary Target (as defined in column J of the mapping inventory spreadsheet), but also that the conditions set
+forth in the Computable ANTLR column are tested. This will just test moving from HL7 V2 -> FHIR. We also need to test 
```

Comment:
nit:

```suggestion
forth in the Computable ANTLR column of the mapping inventory spreadsheet are tested. This will just test moving from HL7 V2 -> FHIR. We also need to test 
```